pcsx2/ImGui/FullscreenUI.cpp

```diff
@@ -4128,8 +4128,8 @@ void FullscreenUI::DrawGraphicsSettingsPage(SettingsInterface* bsi, bool show_ad
 
 	BeginMenuButtons();
 
-	MenuHeading(FSUI_CSTR("Renderer"));
-	DrawStringListSetting(bsi, FSUI_ICONSTR(ICON_FA_PAINTBRUSH, "Renderer"), FSUI_CSTR("Selects the API used to render the emulated GS."), "EmuCore/GS",
+	MenuHeading(FSUI_CSTR("Graphics API"));
+	DrawStringListSetting(bsi, FSUI_ICONSTR(ICON_FA_PAINTBRUSH, "Graphics API"), FSUI_CSTR("Selects the API used to render the emulated GS."), "EmuCore/GS",
```

Comment:
You forgot to modify the Translation String Area at the bottom of this file. Otherwise this string will be untranslatable.

Reply:
> Actually is that section generated somehow? I might be mistaken.

IIRC that section gets generated by `generate_fullscreen_ui_translation_strings.py`, which should be run daily to when producing translation update PRs

Reply:
It looks like it is, just run tools/generate_fullscreen_ui_translation_strings.py.